### PR TITLE
fix(user): allow users to choose notification email. Fixes MA-134

### PR DESCRIPTION
**Pull request by LeonVreling**
Add backend for [#722](https://github.com/AEGEE/frontend/pull/722), fixes issue [MA-134](https://myaegee.atlassian.net/browse/MA-134)

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -123,6 +123,20 @@ const User = sequelize.define('user', {
...
+    primary_email: {
+        type: Sequelize.ENUM('personal', 'gsuite'),
+        allowNull: false,
+        defaultValue: 'personal'
+    },
+    notification_email: {
+        type: Sequelize.VIRTUAL,
+        get() {
+            if (this.primary_email === 'gsuite') {
+                return this.gsuite_id;
```
Comment (serge1peshcoff) [inline models/User.js:135]: do we have a column or field for it?

Comment (LeonVreling) [reply models/User.js:135]: @linuxbandit will include this in his PR for the GSuite wrapper

Comment (LeonVreling) [reply models/User.js:135]: At least according to [the issue of Jira](https://myaegee.atlassian.net/browse/MA-134)

File: models/User.js [hunk 2]
```diff
@@ -123,6 +123,20 @@ const User = sequelize.define('user', {
...
+            }
+            return this.primary_email;
```
Comment (WikiRik) [inline models/User.js:137]: This seems wrong. It should return the email right? Not 'personal'

**Review by WikiRik — CHANGES_REQUESTED** [review]
I might have found the problem with the tests

File: models/User.js (hunk 2 above)
Comment (LeonVreling) [reply models/User.js:137]: Ah yes true, it should be `return this.email`

**Comment by LeonVreling** [comment]
The one issue that codecov still has is that there is no field this.gsuite_id, which is logical as that will be included in the PR of Fabri

**Comment by LeonVreling** [comment]
All I need to do now is add new tests so that Codecov does not complain about not reached all cases of the if-statement

**Comment by WikiRik** [comment]
Okay, so I added a test but it failed of course. We need #184 to be merged first

**Review by WikiRik — APPROVED** [review]
Tests pass, so should be good to me

**Pull request merged**